data/json/effects.json

```diff
@@ -2172,6 +2172,15 @@
     "part_descs": true,
     "base_mods": { "speed_mod": [ -2 ], "pain_min": [ 1 ], "pain_chance": [ 2 ], "pain_max_val": [ -1 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "squeezed",
+    "name": [ "Uncomfortable" ],
+    "desc": [ "You're too fat to wear hard armor on your %s.  It's pretty uncomfortable." ],
```

Comment:
I would avoid using fat here as a descriptor. Something like your extra fat makes it hard to wear armor on your %s. Or being overweight makes it hard to wear hard armor on your %s.